packages/oasgraph/src/resolver_builder.ts

```diff
@@ -331,7 +373,7 @@ export function getResolver ({
  * Attempts to create an object to become an OAuth query string by extracting an
  * OAuth token from the ctx based on the JSON path provided in the options.
  */
-function createOAuthQS (
+function createOAuthQS(
```

Comment:
I believe in our style convention, we have a space between the function name and the parameters. Would you mind reverting these changes? There are a few more in the PR I believe.

Reply:
I will change the vscode linter rules for this project. Hope it is possible, if not I will install jslint.

Reply:
Ok it's done and will be shipped in the next commit.